style.css

```diff
@@ -472,7 +475,13 @@ footer {
 
 .footer-socials li {
   list-style-type: none;
-  padding: 25px 10px;
+  padding: 25px 10px 10px 0;
+}
+
+.home-icon {
+  display: block;
+  height: 25px;
+  margin: 0 auto;
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly add the missing white background and the black line at the bottom of the menu container to match the Figma design. I am not able to see the changes made.

https://user-images.githubusercontent.com/70269507/167770642-bebe860d-4ada-449b-a83d-a1cc9097dc17.mov

Kindly make sure it works on chrome and it is good to deploy the website and see whether the changes are visible.

Reply:
Thanks for the kind words, It appears I'm a bit confuse/challenged on specifics  of implementing this part of the changes. Please I wouldn't know if its possible for you to point me in the direction of what I need to change in my code to effect the required change.

Thank you

Reply:
@zieeco @sja-thedude Hello guys, please I'm lost at what to do to  effect this change as required. The black  line is at the bottom of the footer and I honestly don't know how to go about displaying it in a modal with a height of 100vh. Also I am non-plussed to why I need to display the black line at the bottom of the container which appears to me as the mobile device home menu and not part of the design. Please kindly help with ways forward on this review as I'm stuck on what to do right now and running out of time.

Thank you

Reply:
@kelomo2502 

I understand how you feel but please remember this is a process to help you become a better dev and increase your ability to implement anything you can see from a design, or better still, when you're asked to make a clone of a website, you're expected to add all `**YOU SEE**` and you will meet that in your HTML capstone. 

- What you can do to add the missing background of white and the black line could be achieved below:
NOTE: **_This is just a suggestion, there are many ways to achieve this_**

- Your mobile menu should be placed in a container, and that container will have a white background and a black line
- Use position `fixed, or absolute on the parent`, width and height of 100% and uses flexbox to align its child element below:
```
your_container {
  background: #fff;
  height: 100%;
  width: 100%;
  position: fixed;
  top, bottom, right, left: play with it
}
```

- Your menu now should look like this:
```
your_mobile-content {
  width: 100%;
  height: 90%;
  background: #3c3a39;
}
```

**You can add the remaining part from here**
_I hope you get a view of what to do now and I hope I've been able to unblock you??_

Reply:
NOTE: The easiest way to get a review is by chatting with them on slack, not all of us always quickly check our emails. :stuck_out_tongue_winking_eye: